net-misc/wput/wput-0.6.1.recipe

```diff
@@ -1,10 +1,29 @@
+# WARNING: SUMMARY must be added to recipe here
 DESCRIPTION="wput"
 HOMEPAGE="http://wput.sourceforge.net/"
+COPYRIGHT="2002-2006 Hagen Fritsch"
+LICENSE="GNU GPL v2"
+REVISION="1"
 SOURCE_URI="http://voxel.dl.sourceforge.net/sourceforge/wput/wput-0.6.1.tgz"
 CHECKSUM_MD5="92b41efed4db8eb4f3443c23bf7ceecf"
-REVISION="1"
-STATUS_HAIKU="stable"
-DEPEND=""
+# WARNING: CHECKSUM_SHA256 must be added to recipe here
+
+ARCHITECTURES="x86_gcc2"
```

Comment:
As I mentionned in my other comment, I now think it may not be a good idea to batch-apply the script this way. I think it would be more useful to:
1) Have a list of files that need changes (this pull request does that for now)
2) Have a way to run the script on a single recipe at a time, so we can then manually adjust the result and submit it.

Unless you can make the script good enough that it can test the recipe itself by running Haikuporter and making sure there are no errors left, in that case an automated run may be considered.